package.json

```diff
@@ -20,7 +20,7 @@
     "docs:json": "compodoc -p ./tsconfig.json -e json -d .",
     "storybook": "npm run docs:json && storybook dev -p 6006",
     "build-storybook": "npm run docs:json && storybook build",
-    "test": "jest --maxWorkers=3 --colors --coverage",
+    "test": "jest --colors --coverage",
```

Comment:
Open to feedback with this, I can revert it if needs be.

With `--maxWorkers=3` set, jest was unable to debug locally saying
_"Both --runInBand and --maxWorkers were specified, but these two options do not make sense together. Which is it?"_

Reply:
My experience is that it's runs quicker locally with --runInBand.